### PR TITLE
Changed SRC camera and serial number to use start time instead of sta…

### PR DESCRIPTION
**Pull request by scsides**
Changed HRSC camera and serial number to use start time instead of start clock count

## Description
The HRSC SCR framing camera often takes a series of images a few seconds apart. In some instances the start clock count is identical for all of the images. This was causing the SPICE attached in spiceinit to be inaccurate, and the serial
number for each of the images in the series was also identical making it impossible to use the serial numbers in the control software.

## Related Issue
$4803

## Motivation and Context
#4803 

## How Has This Been Tested?
All tests are awaiting a clean run of CI

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [X] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by scsides** [comment]
In draft until CI passes

**Review by jessemapel — COMMENTED** [review]
Does this impact the tests at all?

Do we need to tick the camera model version for this? I don't think so but I'm not sure.

**Comment by scsides** [comment]
It should affect the tests. That's why I put it into draft mode. 

I've been debating the camera version too. This is a breaking change, so yes it should be. I'll add that unless I see objections.

File: isis/src/mex/objs/MexHrscSrcCamera/MexHrscSrcCamera.cpp [hunk 1]
```diff
@@ -75,18 +75,26 @@ namespace Isis {
...
-    pair<iTime, iTime> startStop = ShutterOpenCloseTimes(et, exposureDuration);
-    setTime(et);
+    // scs 2022-04-16 Changed to use the StartTime instead of the StartClock.
+    // HRSC SRC images taken in a sequence (e.g., Phobos) often have identical StartClock values,
+    // but different StartTime values
+    //QString clockCount = inst["SpacecraftClockStartCount"];
+    //double et = getClockTime(clockCount).Et();
+    //double exposureDuration = (double)inst["ExposureDuration"] / 1000.0;
+    //pair<iTime, iTime> startStop = ShutterOpenCloseTimes(startTime, exposureDuration);
+    //setTime(et);
```
Comment (ladoramkershner) [inline isis/src/mex/objs/MexHrscSrcCamera/MexHrscSrcCamera.cpp]: Could we remove this previous code or are you holding onto it for a particular reason?

File: isis/src/mex/objs/MexHrscSrcCamera/MexHrscSrcCamera.cpp [hunk 2]
```diff
@@ -75,18 +75,26 @@ namespace Isis {
...
-    double exposureDuration = (double)inst["ExposureDuration"] / 1000.0;
 
-    pair<iTime, iTime> startStop = ShutterOpenCloseTimes(et, exposureDuration);
-    setTime(et);
+    // scs 2022-04-16 Changed to use the StartTime instead of the StartClock.
+    // HRSC SRC images taken in a sequence (e.g., Phobos) often have identical StartClock values,
+    // but different StartTime values
+    //QString clockCount = inst["SpacecraftClockStartCount"];
+    //double et = getClockTime(clockCount).Et();
+    //double exposureDuration = (double)inst["ExposureDuration"] / 1000.0;
```
Comment (ladoramkershner) [inline isis/src/mex/objs/MexHrscSrcCamera/MexHrscSrcCamera.cpp]: Do we still need exposureDuration?

File: isis/src/mex/objs/MexHrscSrcCamera/MexHrscSrcCamera.cpp (hunk 1 above)
Comment (scsides) [reply isis/src/mex/objs/MexHrscSrcCamera/MexHrscSrcCamera.cpp]: As a purist it should be removed.

File: isis/src/mex/objs/MexHrscSrcCamera/MexHrscSrcCamera.cpp (hunk 2 above)
Comment (scsides) [reply isis/src/mex/objs/MexHrscSrcCamera/MexHrscSrcCamera.cpp]: It's not used anywhere else, so it is ok to remove

**Review by ladoramkershner — DISMISSED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -34,6 +34,8 @@ release.
 -->
 
 ## [Unreleased]
+- Added check to determine if poles were a valid projection point in ImagePolygon when generating footprint for a map projected image. [#4390](https://github.com/USGS-Astrogeology/ISIS3/issues/4390)
+- Changed the Mars Express HRSC SRC camera and serial number to use the StartTime instead of the StartClockCount  [#4803](https://github.com/USGS-Astrogeology/ISIS3/issues/4803)
```
Comment (jessemapel) [inline CHANGELOG.md]: Move these under an appropriate header

**Review by jessemapel — COMMENTED** [review]
This isn't a breaking change as of right now. We do not consider camera model version ticks to be breaking right now.

**Comment by scsides** [comment]
This change required a change to the serial number for images from the SRC camera. Any existing control networks will need to be fixed. This seems like a breaking change.

**Comment by jessemapel** [comment]
We had similar changes in #4887 and decided it wasn't a break. You should comment on #4792 if you have an opinion about what is and is not API breaking

**Comment by scsides** [comment]
We should go with @jessemapel suggestion to remain consistent.

**Review by ladoramkershner — APPROVED** [review]

**Pull request merged**